docs/en/pid-eaa-presentation.rst

```diff
@@ -0,0 +1,17 @@
+.. include:: ../common/common_definitions.rst
+
+.. _pid-eaa-presentation.rst:
+
+PID/(Q)EAA Presentation
++++++++++++++++++++
+
+This section describes how a remote Relying Party Instance requests to a Wallet Instance the presentation of the PID/EAAs.
+
+In this section the following flows are described:
+
+- :ref:`Remote Flow`, where the User presents a Credential to a remote Relying Party according to `OpenID4VP`_ Draft 20. In this scenario the user-agent and the Wallet Instance can be used in the same device (**Same Device Flow**), or in different devices (**Cross Device Flow**).
```

Comment:
```suggestion
- :ref:`Remote Flow`, where the User presents a Credential to a remote Relying Party according to `OpenID4VP`_. In this scenario the user-agent and the Wallet Instance can be used in the same device (**Same Device Flow**), or in different devices (**Cross Device Flow**).
```

we do not use Draft versions in the main text, we have decided to only define draft versions in their references definition page, see: https://github.com/italia/eid-wallet-it-docs/blob/versione-corrente/docs/common/standards.rst?plain=1#L18

todate, that draft version 23 would became 24.